sample-apps/SampleAppSpec.md

```diff
@@ -143,6 +143,144 @@ These are Key Value pairs to be added on metrics.
 }
 ```
 
+**Expected Metrics Test File**
+
+The Metric names should have "_{{testingID}}" added onto the end of the metric name to ensure the correct metric is grabbed for verification.  The {{Testing ID}} is usually set to the INSTANCE_ID environment variable so an example of a metricName is `cpuUsage + "_" + INSTANCE_ID` or `cpuUsage_c2f47fc1fb1f5bb7`.
```

Comment:
I think you are leaking implementation details of the testing into the spec file. Questions that I think should be answered. 

Does the spec clearly state which labels must be attached to each metric?
Does the spec clearly state how to add a unique id onto each metric name? 

If the answer to both of those is yes then I do not think we need to show how we perform validations. This information and the mustache file pasted below is not relevant to implementation of the sample apps in my opinion.

Reply:
Ok I agree I will add in how to add the unique id onto each of the metric names and remove the rest of the unnecessary details